src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partner = 'wife';
+
+  if (person.sex === 'f') {
+    partner = 'husband';
```

Comment:
According to React logic, it's a bad idea to reassign some variables outside of the state.
Of course, you don't need state here for this static value, but the best way to do what you need is:
```
const partner = person.sex === 'f' ? 'husband' : 'wife';
```
Without any `if`

Reply:
Thanks for advice. :)